scripts-2.0/style/main.scss

```
@@ -1,5 +1,5 @@
-@import 'common';
-@import 'reset-styles';
+@import "common";
+@import "reset-styles";
 
```

```suggestion
@import 'common';
@import 'reset-styles';

```
chyba nie mamy lintera tu podpiętego, ale mamy ustalone, że importy w pojedynczych cudzysłowach